### PR TITLE
Action for automatic release creation

### PR DESCRIPTION
**Pull request by maxbartunik**
New Github action added:
Creates a new release when commit is pushed to main.

### CODE REVIEW

File: .github/workflows/auto-release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
+          prerelease: false
+          title: "PagerBudddy-Server"
+          files: |
```
Comment (keszocze) [inline .github/workflows/auto-release.yml:21]: I think you forgot to add the `test` folder

**Review by keszocze — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-release.yml (hunk 1 above)
Comment (keszocze) [reply .github/workflows/auto-release.yml:21]: I was expecting this to actually show the files that you are including, not the lines above. But I still think that you get the idea.

Comment (maxbartunik) [reply .github/workflows/auto-release.yml:21]: I purposefully removed the test folder. My mental model of a "release" beeing what somebody downloads (or gets from npm/yarn/...) to run the software. Wheras the test cases are only of relevance for development?

I am happy to add the test cases if you disagree...

Comment (keszocze) [reply .github/workflows/auto-release.yml:21]: Well, I checked and they are packaged to the v1 zip file.

Randomly checking the releases of the [Sqlite3 package](https://github.com/TryGhost/node-sqlite3/releases) shows that people *do* ship the tests. If you check the code in the `node_modules` folder, you'll see that many packages provide the tests. So eco-system wise it seems to be fitting. Furthermore,  I'd consider this a nice thing to do. 

I am not fighting you for this though.

Comment (maxbartunik) [reply .github/workflows/auto-release.yml:21]: ;) Neither am I - fine with me

**Pull request merged**